18_migrate_ci_to_gh_actions.md

```diff
@@ -0,0 +1,212 @@
+highlighting any additional information provided beyond the standard enhancement
+template.
+-->
+
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Notes/Constraints/Caveats (optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed (optional)](#infrastructure-needed-optional)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+<!--
+**ACTION REQUIRED:** In order to merge code into a release, there must be an
+issue in [keylime/enhancements] referencing this enhancement and targeting a release**.
+
+For enhancements that make changes to code or processes/procedures in core
+Keylime i.e., [keylime/keylime], we require the following Release
+Signoff checklist to be completed.
+
+Check these off as they are completed for the Release Team to track. These
+checklist items _must_ be updated for the enhancement to be released.
+-->
+
+- [x] Enhancement issue in release milestone, which links to pull request in [keylime/enhancements]
+- [ ] Core members have approved the issue with the label `implementable`
+- [ ] Design details are appropriately documented
+- [ ] Test plan is in place
+- [ ] User-facing documentation has been created in [keylime/keylime-docs]
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+## Summary
+
+The Keylime project's current CI pipeline is implemented using Travis CI's
+toolset. While functional, the team has noticed some shortcomings around
+reliability, performance, and general integration with Github.
+
+In the meantime, Github has rolled out its homegrown Github Actions CI and test
+system to general availability. Project contributors that have used the system
+for other projects have noted its general improvements over Travis, and the
+project would like to transition its existing CI setup over to Actions.
+
+## Motivation
+
+Keylime's current CI pipeline isn't satisfactory to the project's contributors,
+and better alternatives (notably Github Actions) exists. We would like to switch
+to Actions entirely.
+
+In particular, Github Actions is:
+
+- Directly integrated into Github's UI
+- Fast -- jobs start quickly, and most lead time happens at compilation or
+  environment setup
+- Extensible -- there are thousands of actions available for a wide variety of
+  tasks
+- Fits the fork-request-merge model in the way developers expect modern CI to
```

Comment:
Ending with a preposition makes it seem like it's missing a word at the end of this sentence.